src/library_exceptions.js

```diff
@@ -143,6 +143,7 @@ var LibraryExceptions = {
       return {{{ makeGetValue('this.ptr', 'ptrSize', '*') }}};
     };
 
+#if !DISABLE_EXCEPTION_CATCHING
```

Comment:
This entire file should not be included if exceptions are disabled, so this should not be needed - there is a discussion in another issue on this from today?

Reply:
This change is split out from the other change. 

I guess I'll try re-using the existing `_stubs` file.. but I would imagine there was a reason for not doing that way initially.  Lets see.

Reply:
I don't remember the details, but maybe there was something. Anyhow, I think this PR is good as it is, sorry for misunderstanding before.